src/components/AddComment.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { PostComment } from '../../api-post';
+import '../Styling/Addcomment.css'
+export const AddComment = ({ articleId }) => {
+    const [loading, setLoading] = useState(false);
+
+    const username = 'weegembump';
```

Comment:
This would be better placed higher in the app and passed via context so other components can make use of the user.